packages/remix-dev/config/routes.ts

```diff
@@ -54,6 +54,13 @@ export interface DefineRouteOptions {
    * Should be `true` if this is an index route that does not allow child routes.
    */
   index?: boolean;
+
+  /**
+   * An optional unique id string for this route, that will be appended to the file
+   * name e.g: `<file-name>--<id>`. Use this if you need to aggregate two or more
+   * routes with the same route file.
+   */
+  id?: string;
```

Comment:
does the comment above line 28 need to change?

Reply:
@runmoore i don't know it's just a suggest a how to fill a custom ID.

But we are discussing right now about this custom ID and verification about duplicate routes, I'm waiting for @jacob-ebey and @brophdawg11 feedbacks.